plugins/metrics/src/main/java/org/apache/cloudstack/response/HostMetricsResponse.java

```diff
@@ -117,7 +117,7 @@ public void setCpuTotal(final Integer cpuNumber, final Long cpuSpeed, final Doub
 
     public void setCpuUsed(final String cpuUsed, final Integer cpuNumber, final Long cpuSpeed) {
         if (cpuUsed != null && cpuNumber != null && cpuSpeed != null) {
-            this.cpuUsed = String.format("%.2f Ghz", Double.valueOf(cpuUsed.replace("%", "")) * cpuNumber * cpuSpeed / (100.0 * 1000.0));
+            this.cpuUsed = String.format("%.2f Ghz", Double.valueOf(cpuUsed.replace(",",".").replace("%", "")) * cpuNumber * cpuSpeed / (100.0 * 1000.0));
```

Comment:
@Spaceman1984 can you check and fix it at the source, i.e. where the string is created; this may still not work because many EU locales may display something like `10,123.456` as `10.123,456` and with this fix result in something like `10.123.456`

Reply:
Would there be a case where cpu usage is more than 1000% ?

Reply:
This method is taking a percentage value and doing some conversion on it. At the source it's set using DecimalFormat. I have changed my solution to use the same object and formatting so the locale won't be a problem anymore.